Components/CodeBrowser.tsx

```diff
@@ -0,0 +1,20 @@
+import Prism from "prismjs"; //used for syntax highlighting
+import "prismjs/components/prism-python"; //supports syntax highlighting for python only
+import "prismjs/themes/prism-okaidia.css"; //stylizes syntax highlighting
+
+interface CodeBrowserProps {
+  language: string;
+  code: string;
+}
+
+const CodeBrowser: React.FC<CodeBrowserProps> = ({ language, code }) => {
+  if (typeof code !== "string"){
+    return <div>Value is in wrong format</div>
+  } else if (typeof code === 'undefined' || typeof code === null){
+    return <div>Value is undefined or null</div>
+  }
```

Comment:
this branch will never evaluate

Reply:
You're right, I was doing some error checking, thought it was necessary. Getting rid of it.

Reply:
I just ran prettier